tests/test_handler_util.py

```diff
@@ -178,122 +180,490 @@ def test_byte_swapping(self):
-                 [29, 37, 45],
-                 [30, 38, 46],
-                 [31, 39, 47],
-                 [32, 40, 48]],
+        "reference": np.asarray(
+            [
+                [  # Frame 1
+                    [[1, 9, 17], [2, 10, 18], [3, 11, 19], [4, 12, 20], [5, 13, 21]],
+                    [[2, 10, 18], [3, 11, 19], [4, 12, 20], [5, 13, 21], [6, 14, 22]],
+                    [[3, 11, 19], [4, 12, 20], [5, 13, 21], [6, 14, 22], [7, 15, 23]],
+                    [[4, 12, 20], [5, 13, 21], [6, 14, 22], [7, 15, 23], [8, 16, 24]],
+                ],
+                [  # Frame 2
+                    [
+                        [25, 33, 41],
+                        [26, 34, 42],
+                        [27, 35, 43],
+                        [28, 36, 44],
+                        [29, 37, 45],
+                    ],
+                    [
+                        [26, 34, 42],
+                        [27, 35, 43],
+                        [28, 36, 44],
+                        [29, 37, 45],
+                        [30, 38, 46],
+                    ],
+                    [
+                        [27, 35, 43],
+                        [28, 36, 44],
+                        [29, 37, 45],
+                        [30, 38, 46],
+                        [31, 39, 47],
+                    ],
+                    [
+                        [28, 36, 44],
+                        [29, 37, 45],
+                        [30, 38, 46],
+                        [31, 39, 47],
+                        [32, 40, 48],
+                    ],
+                ],
             ]
-        ]),
-        '1frame_1sample': np.asarray(
+        ),
+        "1frame_1sample": np.asarray(
             [1, 2, 3, 4, 5, 2, 3, 4, 5, 6, 3, 4, 5, 6, 7, 4, 5, 6, 7, 8]
         ),
-        '2frame_1sample': np.asarray(
-            [1,   2,  3,  4,  5,  2,  3,  4,  5,  6,  # Frame 1
-             3,   4,  5,  6,  7,  4,  5,  6,  7,  8,
-             25, 26, 27, 28, 29, 26, 27, 28, 29, 30,  # Frame 2
-             27, 28, 29, 30, 31, 28, 29, 30, 31, 32]
+        "2frame_1sample": np.asarray(
+            [
+                1,
+                2,
+                3,
+                4,
+                5,
+                2,
+                3,
+                4,
+                5,
+                6,  # Frame 1
+                3,
+                4,
+                5,
+                6,
+                7,
+                4,
+                5,
+                6,
+                7,
+                8,
+                25,
+                26,
+                27,
+                28,
+                29,
+                26,
+                27,
+                28,
+                29,
+                30,  # Frame 2
+                27,
+                28,
+                29,
+                30,
+                31,
+                28,
+                29,
+                30,
+                31,
+                32,
+            ]
         ),
```

Comment:
Hmm... maybe these very long single-item-per-line listings could be started as 2d arrays that were then flattened for the test.